src/content/smart-bulletins/035.md

```diff
@@ -0,0 +1,68 @@
+number: "No. 035"
+order: 035
+businessline: "Purchase – accounts used for Transit Benefits"
+published: "Sep 1, 2020"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| September 01, 2020 | Implementation of Smart Bulletin #35 |
+
+## Effective Date
+Effective upon issuance and shall remain in force until modified, rescinded, or superseded.
+
+## Business Line(s) Affected
+
+Purchase – accounts used for Transit Benefits
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations that use the GSA SmartPay purchase business line for transit benefit accounts with 
+implementing guidance on submitting statistical and narrative reports as required in Chapter 5.3 of Office of Management and Budget (OMB) Circular A-123, Appendix B, 
+entitled “A Risk Framework for Government Charge Card Programs,” dated August 27, 2019. This Smart Bulletin includes: (1) reporting templates for agency submission of 
+chapter 5.3 reports,(2) reporting expectations for FY2020 and beyond. 
+
+## Summary
+
+OMB memorandum M-17-26 dated June 25, 2017 discontinued agency/organization purchase and travel charge card statistical reporting to OMB. However, OMB in that memorandum also required agencies to maintain that statistical reporting information for their own use in the management of their charge card programs. It further designated GSA’s Center for Charge Card Management (CCCM) as the central repository for the agency charge card statistical data which was previously required to be submitted to OMB. CCCM will post this statistical and narrative data on GSA Interact for agencies/organizations subject to the Chief Financial Officers’ (CFO) Act. CCCM may also post information pertaining to other GSA SmartPay participating agencies/organizations as well. CCCM will work with the GSA SmartPay contractor banks to obtain agency/organization statistical reporting data in an electronic, automated manner to the extent possible. The balance of the statistical data CCCM is unable to directly obtain from contractor banks will need to be provided by the CFO Act agencies/organizations (and other entities as required) to CCCM. CCCM will use GSA Interact, or a successor system, as the central repository for these reports. The Circular also states CCCM “may issue further guidance on both narrative and statistical reporting through the GSA Smart Bulletin process.” This bulletin constitutes that further guidance
+
+### OMB Circular A-123, Appendix B Chapter 5.3.1 – Statistical Reporting –
+Requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2020 agency statistical reporting to CCCM is January 31, 2021. 
```

Comment:
The requested information in the template attached below shall be submitted, at a minimum, on an annual fiscal year basis. Agency-provided information shall be furnished to CCCM no later than 90 calendar days after the end of the Fiscal Year (FY). The due date for agency/organization submission of FY2020 agency statistical reporting to CCCM is January 31, 2021.
The Statistical Reporting requirement applies to all business lines under the GSA SmartPay program; however, the following guidance is for transit benefit issuing programs only. All other agencies should see Smart Bulletin No.030 for reporting guidance. Transit benefit programs must report the following items for each business line:
Count of accounts, as defined by GSA;
Total number of accounts across the charge card program as compared to the prior reporting period;
Total charge card dollars spent; and
Total refunds earned.
Specific requirements for the purchase cards business line are as follows:
Purchase cards used for transit benefits
Number of transit benefit accounts with limits over the statutory limit;
Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA’s Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).